targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F469xI/TARGET_SDP-K1/system_clock.c

```diff
@@ -0,0 +1,275 @@
+/* mbed Microcontroller Library
+* Copyright (c) 2006-2017 ARM Limited
```

Comment:
2019 year

Add SPDX identifier to all new files

Reply:
system_clock.c file was not changed at all. It is used as is. Does the year need to change then?

Reply:
https://github.com/malavikasajikumar/mbed-os/commit/b8d4ad29da1fb4d178d910a3024d9396a5976839